src/redux/books/apiConnect.js

```diff
@@ -0,0 +1,62 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/2t8fqWZuouijX3morClV';
+
+const createBookList = (obj) => {
+  const result = [];
+  const keys = Object.keys(obj);
+
+  keys.forEach((item) => {
+    result.push({
+      author: obj[item][0].author,
+      id: item,
+      category: obj[item][0].category,
+      title: obj[item][0].title,
+    });
+  });
+  return result;
+};
+
+export const getBooks = createAsyncThunk(
+  'books/getBooks',
+  async () => {
+    const response = await fetch(`${baseUrl}/books`, {
+      method: 'GET',
```

Comment:
# Optional: 
- The requirements stated that **styling is not Required**. Please follow the requirements.
- 
![Capture](https://user-images.githubusercontent.com/63972670/208184978-5ebf404a-8f1b-4935-8452-fd3d1d2200d1.PNG)